packages/devtools_app/lib/src/ui/service_extension_widgets.dart

```diff
@@ -893,8 +894,8 @@ class ServiceExtensionRichTooltip extends StatelessWidget {
               Align(
```

Comment:
only show this align widget if both description.tooltipUrl != null and description.gaScreenName != null

Reply:
done